redirects-www.conf

```diff
@@ -45,8 +45,10 @@ rewrite ^/info/filing.shtml /help-candidates-and-committees/ redirect;
 rewrite ^/info/report_dates_2017.shtml https://transition.fec.gov/info/report_dates_2017.shtml redirect;
 rewrite ^/pdf/eleccoll.pdf https://transition.fec.gov/pdf/eleccoll.pdf redirect;
 rewrite ^/elecfil/electron.shtml https://transition.fec.gov/elecfil/electron.shtml redirect;
-rewrite ^/efiling /help-candidates-and-committees/electronic-filing/ redirect;
-rewrite ^/efiling/ /help-candidates-and-committees/electronic-filing/ redirect;
+rewrite ^/efiling /help-candidates-and-committees/filing-reports/electronic-filing/ redirect;
```

Comment:
Thanks for catching that @LindsayYoung. You are correctly, there is no need for the redirects with trailing slashes. I'll remove them.